packages/fxa-settings/src/pages/Signin/SigninTokenCode/container.tsx

```diff
@@ -37,22 +35,25 @@ const SigninTokenCodeContainer = ({
     integration
   );
 
-  // reads from cache if coming from /signin
-  const { data: totpData, loading: totpLoading } =
-    useQuery<TotpStatusResponse>(GET_TOTP_STATUS);
+  const [totpEnabled, setTotpEnabled] = useState<boolean>(false);
+  const account = useAccount();
+
+  useEffect(() => {
+    const getTotpStatus = async () => {
+      const totpData = await account.hasTotpAuthClient();
```

Comment:
We have generally avoided using the `useAccount` hook in non-settings pages (to avoid over fetching). What was the issue with the gql query?

Reply:
Yea, unfortuantely I was getting `Must verify` error from gql. There is probably a bug somewhere with how its handling it but wasn't very obivous what was going on. I'll add a note here why there is an exception.

Reply:
@vbudhram instead of using `useAccount` which tries to fetch all account data, can we reference `authClient` directly? We do this in a few other places [like here](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/pages/Signin/container.tsx#L277) [or here](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/pages/Signin/CompleteSignin/container.tsx#L42) and then cache the result. Otherwise we'll want a follow up to prevent the extra data fetching here.

Reply:
Updated to use `authClient`